.github/workflows/check-mirrors.yml

```
@@ -0,0 +1,83 @@
+#
+# This workflow checks if a mirror is offline or out-of-dated.
+#
+name: Check Mirrors
+on:
+  # uncomment the 'pull_request' line to enable testing in PRs
+  pull_request:
```

Disable the workflow in PR:
```suggestion
  # pull_request:
```